### PR TITLE
php2branch >tl1revbranch

### PR DESCRIPTION
**Pull request by php-2**
h

### CODE REVIEW

File: portfolio/ifintech/v1/ierp/ierp-7.0.0/htdocs/custom/extintegration/sql/general/data.sql [hunk 1]
```diff
@@ -13,36 +13,251 @@ VALUES (1,'2018-02-15 12:37:13.000','2018-02-15 12:37:13.000','1','Cancel','3','
...
-INSERT INTO llx_sys_module (rowid, code, title, status)         VALUES (60, 'MOD-ISCADA',   'iScadaModule',     '1');
-INSERT INTO llx_sys_module (rowid, code, title, status)         VALUES (61, 'MOD-IEML',     'iEmailModule',     '1');
-INSERT INTO llx_sys_module (rowid, code, title, status)         VALUES (62, 'MOD-IEML',     'iSmsModule',       '1');
-INSERT INTO llx_sys_module (rowid, code, title, status)         VALUES (63, 'MOD-IBUDG',    'iBudgModule',      '1');
-INSERT INTO llx_sys_module (rowid, code, title, status)         VALUES (64, 'MOD-IEXP',     'iExportModule',    '1');
-INSERT INTO llx_sys_module (rowid, code, title, status)         VALUES (65, 'MOD-MANUF',    'iManufModule',     '1');
-INSERT INTO llx_sys_module (rowid, code, title, status)         VALUES (66, 'MOD-IDASH',    'iDashModule',      '1');
-INSERT INTO llx_sys_module (rowid, code, title, status)         VALUES (67, 'MOD-ITASKS',   'iTasksModule',     '1');
+INSERT INTO llx_sys_module (rowid, code, title, status, doli_module_name)         VALUES (1,  'MOD-CALD',     'CalendarModule',   '0', 'Calender');
```
Comment (hythamsoliman81) [inline portfolio/ifintech/v1/ierp/ierp-7.0.0/htdocs/custom/extintegration/sql/general/data.sql]: usually the status is the last filed at the table
move doli_module_name to be before status

File: portfolio/ifintech/v1/ierp/ierp-7.0.0/htdocs/custom/extintegration/sql/general/data.sql [hunk 2]
```diff
@@ -13,36 +13,251 @@ VALUES (1,'2018-02-15 12:37:13.000','2018-02-15 12:37:13.000','1','Cancel','3','
...
+INSERT INTO llx_sys_module (rowid, code, title, status, doli_module_name)         VALUES (61, 'MOD-IEML',     'iEmailModule',     '1', 'iemail');
+INSERT INTO llx_sys_module (rowid, code, title, status, doli_module_name)         VALUES (62, 'MOD-IEML',     'iSmsModule',       '1', 'isms');
+INSERT INTO llx_sys_module (rowid, code, title, status, doli_module_name)         VALUES (63, 'MOD-IBUDG',    'iBudgModule',      '1', 'ibudget');
+INSERT INTO llx_sys_module (rowid, code, title, status, doli_module_name)         VALUES (64, 'MOD-IEXP',     'iExportModule',    '1', 'iexport');
+INSERT INTO llx_sys_module (rowid, code, title, status, doli_module_name)         VALUES (65, 'MOD-MANUF',    'iManufModule',     '1', 'imanufacture');
+INSERT INTO llx_sys_module (rowid, code, title, status, doli_module_name)         VALUES (66, 'MOD-IDASH',    'iDashModule',      '1', 'idash');
+INSERT INTO llx_sys_module (rowid, code, title, status, doli_module_name)         VALUES (67, 'MOD-ITASKS',   'iTasksModule',     '1', 'itasks');
 --end new code 
+--prodcut_links
+INSERT INTO llx_sys_idash_form (rowid, modid, code, title, type_id, status_id)
```
Comment (hythamsoliman81) [inline portfolio/ifintech/v1/ierp/ierp-7.0.0/htdocs/custom/extintegration/sql/general/data.sql]: don't make it on 2 lines! make each insert statement in one line.

Comment (hythamsoliman81) [inline portfolio/ifintech/v1/ierp/ierp-7.0.0/htdocs/custom/extintegration/sql/general/data.sql]: for all rows

modid should be mod_id
status_id  should be status

Comment (hythamsoliman81) [inline portfolio/ifintech/v1/ierp/ierp-7.0.0/htdocs/custom/extintegration/sql/general/data.sql]: if rowid is auto incremental why do you add to insert statement?

File: portfolio/ifintech/v1/ierp/ierp-7.0.0/htdocs/custom/extintegration/sql/general/data.sql (hunk 1 above)
Comment (php-2) [reply portfolio/ifintech/v1/ierp/ierp-7.0.0/htdocs/custom/extintegration/sql/general/data.sql]: ok i will move it

File: portfolio/ifintech/v1/ierp/ierp-7.0.0/htdocs/custom/extintegration/sql/general/data.sql (hunk 2 above)
Comment (php-2) [reply portfolio/ifintech/v1/ierp/ierp-7.0.0/htdocs/custom/extintegration/sql/general/data.sql]: ok

Comment (php-2) [reply portfolio/ifintech/v1/ierp/ierp-7.0.0/htdocs/custom/extintegration/sql/general/data.sql]: i will change it

**Pull request merged**